gpep517/__main__.py

```diff
@@ -84,12 +85,18 @@ def override_writestr(self, zinfo_or_arcname, data,
 
 
 @contextlib.contextmanager
-def patch_sysconfig(sysroot: str):
+def patch_sysconfig(sysroot: Path,
+                    prefix_from: typing.Optional[str],
+                    prefix_to: typing.Optional[str],
+                    ):
     stdlib_path = Path(sysconfig.get_path("stdlib"))
     if not stdlib_path.is_absolute():
         raise RuntimeError(f"stdlib path {stdlib_path} is not absolute")
-    data_paths = list(
-        (sysroot / stdlib_path.relative_to("/")).glob("_sysconfigdata_*.py"))
+    stdlib_path = Path("/", prefix_to or "",
+                       str(stdlib_path).removeprefix(prefix_from or "")[1:])
```

Comment:
I wonder if we could use `relative_to()` for this instead. It should be safer in terms of normalization, I think.

Reply:
It's a bit awkward but okay.

Reply:
I meant instead of `removeprefix()`.

Reply:
Ah, of course, should have thought of that. Thanks. Fixed now.